go.mod

```diff
@@ -8,7 +8,9 @@ require (
 	github.com/go-logr/logr v0.4.0
 	github.com/jackc/pgx/v4 v4.11.0
 	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20211209195740-297c4b4e4fbc
+	github.com/open-cluster-management/multicloud-operators-channel v1.0.1-0.20201120143200-e505a259de45
```

Comment:
need to pay attention to versions. we need to reference the versions that are compatible with ACM 2.4. 
this is from 2020 and I believe a newer version should be used.